src/garage/np/algos/off_policy_rl_algorithm.py

```diff
@@ -94,6 +95,7 @@ def train(self, runner):
                 last_return = self.train_once(runner.step_itr,
                                               runner.step_path)
                 if cycle == 0 and self._buffer_prefilled:
```

Comment:
Why not just remove the check for `self._buffer_prefilled`? That would simpler, and would cause the example test to actually catch failures in `_obtain_evaluation_samples`.